### ISSUE TITLE
Bias in Layer 2 set to 1 in finetune_flickr_style

### ISSUE THREAD

**Issue by EvanZ** [BVLC/caffe#3668]
Thought this was curious (from `train_test_val.prototxt`):

`layer {
  name: "conv2"
  type: "Convolution"
  bottom: "norm1"
  top: "conv2"
  param {
    lr_mult: 1
    decay_mult: 1
  }
  param {
    lr_mult: 2
    decay_mult: 0
  }
  convolution_param {
    num_output: 256
    pad: 2
    kernel_size: 5
    group: 2
    weight_filler {
      type: "gaussian"
      std: 0.01
    }
    bias_filler {
      type: "constant"
      value: 1
    }
  }`

Any reason why the bias is set to 1 instead of 0 as in the other layers? I've been having some issues fine tuning a different data set using this is as a starting point, and while I doubt this is the main culprit, I figured I would mention it here anyway.

**Comment by seanbell** [comment]
Since the flickr style example is fine-tuning from the ImageNet weights, the bias initialization parameter is not actually used.  This parameter has no effect.  If you're starting from scratch (random weights), then training may not work for a variety of reasons (too little data, labels are too hard, the bias of 1 for some layer).

In general, setting bias to 1 is sometimes done to make training easier in the beginning.  It means that less of the signal  is clipped at the time of initialization.  However, you can't set it too high, or else none of the signal gets clipped and then the layer is no longer nonlinear.

**Comment by ginobilinie** [comment]
In my experience, if you train a model with bias filler at 1, a big chance is that your model will hardly converge. So set to 0 when training is my suggestion.

**Comment by shelhamer** [comment]
From https://github.com/BVLC/caffe/blob/master/CONTRIBUTING.md:

> _Please do not post usage, installation, or modeling questions, or other requests for help to Issues._
Use the [caffe-users list](https://groups.google.com/forum/#!forum/caffe-users) instead. This helps developers maintain a clear, uncluttered, and efficient view of the state of Caffe.
